vocabularies/Hierarchy.json

```diff
@@ -140,6 +155,118 @@
         ],
         "$ReturnType": { "$Collection": true, "$Type": "Edm.EntityType" }
       }
+    ],
+    "Template_CreateAction": [
+      {
+        "$Kind": "Action",
+        "$EntitySetPath": "HierarchyNodes",
+        "$IsBound": true,
+        "@Common.Experimental": true,
+        "@Core.Description": "Template for actions that create a new node and are named in [`RecursiveHierarchy/CreateAction`](#RecursiveHierarchyType)",
+        "$Parameter": [
+          { "$Name": "HierarchyNodes", "$Collection": true, "$Type": "Edm.EntityType" },
+          { "$Name": "HierarchyQualifier", "$Type": "Aggregation.HierarchyQualifier" },
+          {
+            "$Name": "Entity",
+            "$Type": "Edm.EntityType",
+            "@Core.Description": "The entity to become a new node",
+            "@Core.LongDescription": "If the entity exists already without a node identifier for the recursive hierarchy,\n            it is upserted with PATCH semantics.\n            It is an error if the entity exists already with a node identifier for the recursive hierarchy."
+          },
+          {
+            "$Name": "Node",
+            "$Type": "Edm.PrimitiveType",
+            "@Core.Description": "The node identifier, defaults to the node property of the `Entity`",
+            "@Core.LongDescription": "To insert an existing entity with key property `ID` as a new node with server-determined node identifier,\n            provide the parameter `\"Entity\": {\"ID\": \"...\"}` and omit `Node`.",
```

Comment:
What is the purpose of "Node"? When would it ever be provided?

Reply:
You can always specify the node identifier property of `Entity` instead. But the `Node` parameter offers convenience if you don't know the name of/path to that property and only know the value.

Reply:
But then `Entity` still needs to be provided?
I would assume that our client knows where the node property is in `Entity`.